### ISSUE TITLE
Live player closing on phone when debugging

### ISSUE THREAD

**Issue by syvers147** [xamarin/Xamarin.Forms#1961]
### Description

I have been following the xamarin book supplied by xamarin. The black cats application is causing the live player to simply shut down on my phone with no error messages.

Unless the code in the book is incorrect there must be a bug when accessing resources.

### Steps to Reproduce

1.  Down load Black Cats Solution from your xamarin book here

https://developer.xamarin.com/guides/xamarin-forms/creating-mobile-apps-xamarin-forms/

2. Run via live player wifi connection to mobile phone 

3. Live player switches off when this line is run

     using (Stream stream = assembly.GetManifestResourceStream(resource))
 

### Expected Behavior
The text file should open 

### Actual Behavior
The live player switches off completely and phone goes to main menu screen, no error messages just a complete failure

### Basic Information

TO be honest if you take the project from your book you will have all the same settings as me

wei Honor. However phone is fine, it is only the project I refer to that is causing the error so far. I am guessing it cannot locate the file but even if that is the case why is there no detail of any exception? I even put in a try catch block but it does not give any details and does not catch an error before it fails

### Reproduction Link

A link to the solution used in your book

https://github.com/xamarin/xamarin-forms-book-samples/tree/master/Chapter04/BlackCat

**Comment by MelbourneDeveloper** [comment]
We can't get Live Player to run at all on Android.

**Comment by syvers147** [comment]
If you comment all the code inside the using block it might open in live player. What I did to confirm where the error is coming from is first comment out the using block of code and run in live player, this worked and opened a blank page (you could stick a label in or something to show it really is working).

 Next I copied the code to get the stream into its own line of code so just simply:

Stream stream = assembly.GetManifestResourceStream(resource)

whilst leaving the rest commented.

This time the live player did not work again.

Hope that helps.
Paul

**Comment by samhouts** [comment]
I believe this is a limitation of the LivePlayer at this time. Thank you!

**Comment by syvers147** [comment]
Sorry, what is the limitation?

Live player works on Android so is the limitation when trying to access a file? It says in the notes that your side cannot even get Live Player working so I don't know if you mean it wont work altogether or just that it is when accessing files.

I cannot believe you have just dismissed the whole issue in that manner.  I am speaking to Xamarin employees here aren't I ?

Thanks

**Comment by samhouts** [comment]
I think there might be some confusion. We (Microsoft) did not say that the LivePlayer doesn't work on Android. It does. It just has a set of known limitations at this time, as it is still in development and is released in preview. You can visit the documentation [here](https://developer.xamarin.com/guides/cross-platform/live/limitations/) to view other known limitations. Thanks!

**Comment by syvers147** [comment]
Thanks. Now I have the link and your edit I can see why it does not work. I had no idea it was a preview release, I thought the preview part was to do with using it to "preview" your app!

I have managed to get the app working in an emulator, although that exact same emulator would not run it previously hence I tried the live player. I spent 2 days trying to get it running in the emulator before I tried the live player, then today I cleaned the solution, deleted bin and debug folders and some other files and it built....it is all a bit on and off this xamarin stuff at the moment I am finding. I think I shall give it a miss for a while and maybe try again after a few more releases.

Thanks again
Paul
